### DISCUSSION TITLE
[help] tar_visnetwork plot size

### DISCUSSION THREAD

**Discussion by rsangole** [ropensci/targets#1330] [Help] [answered]
### Help

- [X] I understand and agree to <https://books.ropensci.org/targets/help.html>.

### Description

Hello,

When using `tar_visnetwork(T)` in VSCode, the network remains restricted inside the red rectangle within the Viewer window. Is there a way to change the size of this red window? Large networks are hard to visualize within those constraints, especially when a ton of screen space is available.

Thanks!

![image](https://github.com/user-attachments/assets/38680134-9582-424d-aff8-718846bb4968)

**Comment by wlandau** [comment, marked as answer]
You could try piping `tar_visnetwork()` into `visNetwork::visOptions()`:

```r
targets::tar_visnetwork() |>
  visNetwork::visOptions(width = "800px", height = "800px")
```

**Reply by rsangole** [reply]
Perfect, thanks!
